stable/prometheus/values.yaml

```diff
@@ -184,6 +184,19 @@ configmapReload:
     tag: v0.1
     pullPolicy: IfNotPresent
 
+  ## Additional configmap-reload container arguments
+  ##
+  extraArgs: {}
+
+  ## Additional configmap-reload mounts
+  ##
+  extraConfigmapMounts: []
+    # - name: prometheus-alerts
+    #   mountPath: /etc/alerts.d
+    #   configMap: prometheus-alerts
```

Comment:
I may not fully understand how the code works, but it looks like this line from the example is not being parsed by the actual server-deployment.yaml, correct?

Reply:
Good spot you're absolutely correct! I've missed the bit under volumes e.g. [here](https://github.com/echo-health/charts/blob/master/stable/prometheus/templates/server-deployment.yaml#L148-L152). I'll add that in.